tests/unit/aiplatform/test_metadata.py

```diff
@@ -92,7 +92,9 @@ def get_context_mock():
             display_name=_TEST_EXPERIMENT,
             schema_title=constants.SYSTEM_EXPERIMENT,
             schema_version=constants.SCHEMA_VERSIONS[constants.SYSTEM_EXPERIMENT],
-            metadata={constants.EXPERIMENT_DELETED_KEY: constants.EXPERIMENT_DELETED_VALUE},
+            metadata={
```

Comment:
this will probably break

Reply:
oops, thanks for catching this. Updated now.